fdk_rdf_parser_service/service/parser_service.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import List
+
+import aiohttp
+from fdk_rdf_parser_service.model.parse_result import ParsedCatalog, RdfParseResult
+from fdk_rdf_parser_service.model.rabbit_report import FdkIdAndUri, RabbitReport
+
+
+async def handle_reports(reports: List[RabbitReport]):
+    """Handle reports."""
+    results: List[RdfParseResult] = [await handle_report(report) for report in reports]
+
+    # Sorter i vellykkede og mislykkede parsinger
```

Comment:
Er ikke akkurat det som skjer her, men det kommer vel senere?

Reply:
Stemmer, "note to self" for neste steg